packages/components/ovh-shell/src/client/api.ts

```diff
@@ -1,19 +1,29 @@
 import ShellClient from './shell-client';
 
-export default function exposeApi(shell: ShellClient) {
+export default function exposeApi(shellClient: ShellClient) {
   return {
     i18n: {
       getLocale: () =>
-        shell.invokePluginMethod({
+        shellClient.invokePluginMethod({
           plugin: 'i18n',
           method: 'getLocale',
         }),
+      onLocaleChange: (callback: CallableFunction) =>
+        shellClient.addEventListener('locale-change', callback),
```

Comment:
Maybe it should be a good idea to prefix the event name with the name of the plugin?
Something like `i18n:locale-change`?
I searched for best practice and I found this article: https://gomakethings.com/custom-event-naming-conventions-in-vanilla-js/